### PR TITLE
Housekeeping: Remove outdated documentation 

### PR DESCRIPTION
**Pull request by craigcoles**
# Overview

* Remove outdated documentation
* Move the new documentation into the `docs` folder

---

###  ⚠️ Vercel will need to be reconfigured once this PR is merged ⚠️

---

**Checklist**:

- [x] Documentation
- [ ] Tests **N/A**
- [ ] Code complete **N/A**
- [ ] Changeset **N/A**

### CODE REVIEW

File: docs/docs/guides/buying-and-selling.mdx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+const tokenId = 0;
+
+const nft = sdk.getNFT(contractAddress);
+await nft.buyWithCommission(tokenId, commissionReceiverAddress);
+```
+</TabItem>
+
+<TabItem value="react" label="React">
+
+```tsx
```
Comment (Sam-Archie) [inline docs/docs/guides/buying-and-selling.mdx:86]: Should we just remove this for now?

File: docs/docs/guides/metadata.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+    release_type
+  }
+}
+```
+
+### Querying a single token
+
+```graphql
+{
+  token(id: "0xc922b16f4e9d299fd5fc5b8375928fa761484042") {
```
Comment (Sam-Archie) [inline docs/docs/guides/metadata.md:129]: Is it worth highlighting here that the subgraph explicitly wants addresses formatted in lower case??? I can say from personal experience I have lost a fair bit of time trying to understand what is was about my query that was incorrect before looking at the casing.

File: docs/docs/quickstart/javascript.mdx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+const sdk = new OpenFormatSDK({
+  network: 'mumbai',
+  signer: '0x...',
+});
+```
+
+:::danger private keys
+
+**NEVER** pass a private key when initialising the SDK in the client, as this is something that you **DO NOT** want to expose to the world.
```
Comment (Sam-Archie) [inline docs/docs/quickstart/javascript.mdx:47]: Might be worth looking at re-wording this in the future (I appreciate that this is my copy)

File: docs/docs/quickstart/react.mdx [hunk 4]
```diff
@@ -0,0 +1,243 @@
+---
+title: React
+hide_title: true
+---
+
+<h1 style={{ display: 'flex', gap: "2rem", alignItems: 'center' }}>Quickstart: React <img alt="npm" src="https://img.shields.io/npm/v/@simpleweb/open-format-react"/></h1>
+
+The React SDK adds a small abstraction over the [Javascript SDK](./javascript) but this simplifies interacting with Open Format even further.
+
```
Comment (Sam-Archie) [inline docs/docs/quickstart/react.mdx:9]: Is it worth highlighting that this is only a requirement with create react app?? I spun up a next app today and there were no issues. I appreciate this is a small point and probably a large piece of work to be doing so late in the day.

File: docs/docs/quickstart/react.mdx [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+You can allow people to connect their wallets using the `<ConnectButton />` component (which uses [Onboard](https://www.blocknative.com/onboard)) and the `useWallet` hook to get the connection state and the wallet itself if required.
+
+```tsx
+import { ConnectButton, useWallet } from '@simpleweb/open-format-react';
+
+function MyComponent() {
+  const { isConnected, wallet } = useWallet();
+
+  return (
+    <>
```
Comment (Sam-Archie) [inline docs/docs/quickstart/react.mdx:185]: This is quite an important piece of the puzzle, when getting started using the provider you must be connected before being able to use any of the functions extracting from the hooks, is this worth highlighting?

**Review by Sam-Archie — APPROVED** [review]
A few minor things that might be worth discussing tomorrow in order to log or action at a later date, overall I think the docs look really sharp.

File: docs/docs/guides/buying-and-selling.mdx (hunk 1 above)
Comment (tinypell3ts) [reply docs/docs/guides/buying-and-selling.mdx:86]: Let's remove it.

File: docs/docs/guides/metadata.md (hunk 2 above)
Comment (tinypell3ts) [reply docs/docs/guides/metadata.md:129]: Yes, I will write a ticket for it.

File: docs/docs/quickstart/javascript.mdx (hunk 3 above)
Comment (tinypell3ts) [reply docs/docs/quickstart/javascript.mdx:47]: Let's leave this for now. I will add to backlog.

File: docs/docs/quickstart/react.mdx (hunk 5 above)
Comment (tinypell3ts) [reply docs/docs/quickstart/react.mdx:185]: I don't think this worth highlighting as connecting your wallet to a dApp is common practice.

**Review by tinypell3ts — APPROVED** [review]

**Pull request merged**